packages/http-client-csharp/emitter/src/lib/operation-converter.ts

```diff
@@ -210,7 +210,17 @@ function loadLongRunningOperation(
   if (method.kind !== "lro") {
     return undefined;
   }
-
+  /* Remove this workaround when https://github.com/Azure/typespec-azure/issues/1538 is resolved */
+  if (
+    method.__raw_lro_metadata.finalEnvelopeResult &&
+    method.__raw_lro_metadata.finalEnvelopeResult !== "void" &&
```

Comment:
can we remove this line?

Reply:
No. finalEnvelopeResult is a Union ("void"|sdkModel), we only need to handle when it is model.